marimo/_ast/visitor.py

```diff
@@ -365,7 +365,7 @@ def visit_Call(self, node: ast.Call) -> None:
             elif isinstance(first_arg, ast.JoinedStr):
                 sql = normalize_sql_f_string(first_arg)
 
-            if isinstance(sql, str) and DependencyManager.has_duckdb() and sql:
+            if isinstance(sql, str) and DependencyManager.duckdb.has() and sql:
```

Comment:
Should this be `has_at_version`? `extract_statements` isn't available in some earlier versions of `duckdb`.

Reply:
yea i can add it here too